### PR TITLE
Initial draft of 1.5.5 release notes

### PR DESCRIPTION
**Pull request by George-Aeillo**
Initial draft of 1.5.5 release notes

### CODE REVIEW

File: content/en/docs/releasenotes/_index.md [hunk 1]
```diff
@@ -63,6 +63,14 @@ draft: false
 
 
 ## Previous Release Information
+## v1.5.5
+### Fixes:
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:67]: -> (In Previous Releases, the headings under the Release Number are just bolded, not head 3's.)

File: content/en/docs/releasenotes/_index.md [hunk 2]
```diff
@@ -63,6 +63,14 @@ draft: false
...
+- Fixed issues regarding Rancher certificates that were out of sync or expired.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:68]: -> Fixed issues with Rancher certificates...

File: content/en/docs/releasenotes/_index.md [hunk 3]
```diff
@@ -63,6 +63,14 @@ draft: false
...
+- Fixed issues regarding certificates in Multicluster environments.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:69]: -> Fixed issues with certificates in multicluster environments.

File: content/en/docs/releasenotes/_index.md [hunk 4]
```diff
@@ -63,6 +63,14 @@ draft: false
...
+- Fixed issue where Istio was incorrectly waiting for disabled deployments.
+- Fixed upgrade issues in the CLI tool.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:71]: -> the command-line tool (CLI).

**Review by rosemarymarano — COMMENTED** [review]
Few edits

File: content/en/docs/releasenotes/_index.md (hunk 2 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:68]: Thanks!

File: content/en/docs/releasenotes/_index.md (hunk 3 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:69]: Thanks!

File: content/en/docs/releasenotes/_index.md (hunk 4 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:71]: Thanks!

File: content/en/docs/releasenotes/_index.md [hunk 5]
```diff
@@ -63,12 +63,12 @@ draft: false
...
 
 ## Previous Release Information
-## v1.5.5
-### Fixes:
-- Fixed issues regarding Rancher certificates that were out of sync or expired.
-- Fixed issues regarding certificates in Multicluster environments.
+### v1.5.5
+**Fixes**:
+- Fixed issues with Rancher certificates that were out of sync or expired.
+- Fixed issues with certificates in Multicluster environments.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:69]: -> multicluster (lowercase)

**Review by rosemarymarano — COMMENTED** [review]
Just two more minor fixes needed.

File: content/en/docs/releasenotes/_index.md (hunk 5 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:69]: Thanks!

File: content/en/docs/releasenotes/_index.md [hunk 6]
```diff
@@ -63,12 +63,12 @@ draft: false
...
 - Fixed issue where Istio was incorrectly waiting for disabled deployments.
-- Fixed upgrade issues in the CLI tool.
+- Fixed upgrade issues in the command-line tool (CLI) tool.
```
[the remark this answers is not in the document]
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md]: Thanks!

**Review by rosemarymarano — APPROVED** [review]

**Pull request merged**